api/src/main/java/org/openmrs/module/appointments/model/Appointment.java

```diff
@@ -250,6 +251,14 @@ public Date getDateCreated() {
         return dateCreated;
     }
 
+    public Date getDateAppointmentScheduled() {
```

Comment:
Actually, sorry, I should have comments yesterday, but can we add a comment here explaining the usage of this field (ie "the date appointment was scheduled", not "the date the appointment is scheduled for", since I think it has the potential to be confusing)

Reply:
@mogoodrich please re-review

Reply:
This looks good to me now, (the comment has been added above in lines 27-30